### ISSUE TITLE
[Client] Errors are just logged in background

### ISSUE THREAD

**Issue by dcransac** [denkbar/djigger#19]
Should be presented to the user through the GUI

**Comment by jeromecomte** [comment]
I've replaced all the system outputs by logging. Errors should yet at least be found in the logs. I  would suggest that we close this issue and reopen it for the specific cases where we want them explicitly to be shown to the user. There's unfortunately no way to solve this in a generic way actually.

**Comment by dcransac** [comment]
Yeah I guess that has to do with the rigidity of swing listeners..uh.

How about this though : we dedicate a small area of the GUI to displaying information (INFO, WARNING, ERROR for example), like just a narrow horizontal frame at the bottom of the screen, and then we implement a sort of static queue/worker system, that takes of displaying any error message, and you could write into that queue from anywhere in the code. I know it doesn't sound sexy but that would work right?

Either that or we just implement something cleaner from the get go in the JS client version.

@jsclient

**Comment by jeromecomte** [comment]
Nothing to do with swing but with my own code that is sometimes catching exceptions and re-throwing them :S 

For this reason this issue has to be solve case by case. Only question is: do we open an issue for each case or use this issue for all the potential cases?

**Comment by dcransac** [comment]
We'll keep this issue open and address the problem either completely or we'll at least start with the most common problems that confuse the use.
